config/router.config.js

```diff
@@ -14,71 +19,82 @@ module.exports = [
     name: 'password',
     component: './PasswordHandler',
   },
+  {
+    path: '/exception/403',
+    name: 'exception-403',
+    component: './Exception/403',
+  },
+  {
+    path: '/exception/404',
+    name: 'exception-404',
+    component: './Exception/404',
+  },
+  {
+    path: '/exception/500',
+    name: 'exception-500',
+    component: './Exception/500',
+  },
+  {
+    path: '/share/:id',
+    name: 'share',
+    component: './SessionPlaceholder',
+  },
   {
     path: '/',
     component: '../layouts/BasicLayout',
-    Routes: ['./src/pages/PrivateRoute/index.js'],
     routes: [
       {
         path: '/',
-        name: 'dashboard',
-        icon: 'dashboard',
-        component: './Controllers',
-      },
-      {
-        path: '/profile',
-        name: 'profile',
-        component: './Profile',
-      },
-      {
-        path: '/results',
-        name: 'results',
-        component: './Results',
-      },
-      {
-        path: '/summary',
-        name: 'summary',
-        component: './Summary',
-      },
-      {
-        path: '/comparison-select',
-        name: 'comparison-select',
-        component: './ComparisonSelect',
-      },
-      {
-        path: '/comparison',
-        name: 'comparison',
-        component: './RunComparison',
-      },
-      {
-        path: '/search',
-        name: 'search',
-        component: './Search',
-      },
-      {
-        path: '/explore',
-        name: 'explore',
-        component: './Explore',
-      },
-      {
-        path: '/exception/403',
-        name: 'exception-403',
-        component: './Exception/403',
-      },
-      {
-        path: '/exception/404',
-        name: 'exception-404',
-        component: './Exception/404',
-      },
-      {
-        path: '/exception/500',
-        name: 'exception-500',
-        component: './Exception/500',
+        name: 'publicview',
+        component: './PublicView',
       },
       {
-        path: '/share/:id',
-        name: 'share',
-        component: './SessionPlaceholder',
+        path: '/private',
+        Routes: ['./src/pages/PrivateRoute/index.js'],
+        routes: [
+          {
+            path: '/private',
```

Comment:
Will it be okay to add a new route before each of our routes i.e. `private` followed by the regular routes. When users are authenticated, we can append their username to the route as a wildcard or something, for example:
the route when I am authenticated would become `http://<host>/{aquib}/controllers`. Idk if that would be better but we can do something similar.